Strategic-Initiatives.md

```diff
@@ -14,7 +14,6 @@ and have the support needed.
 | Initiative        | Champion                                                  | Links                                                            |
 |-------------------|-----------------------------------------------------------|------------------------------------------------------------------|
 | CSM            | [Trent Hare][thareusgs]                                    | https://github.com/USGS-Astrogeology/CSM-CameraModel |
-| SPICE Modularization           | [Jay Laura][jlaura] + [Jesse Mapel][jessemapel]                                | https://github.com/USGS-Astrogeology/SpiceRefactor |
```

Comment:
This is an ISIS initiative and will be added under the ISIS Strategic Initiatives.